_data/conferences.yml

```
@@ -23,6 +23,13 @@
 #
 ##########################
 
+- name: App Builders
+  link: https://appbuilders.ch
+  start: 2020-05-11
+  end: 2020-05-12
+  location: 🇨🇭 Lugano, Switzerland
+  cocoa-only: true
```

They are not only about apple technologies, could you change it? Thanks!
```suggestion
  cocoa-only: false
```

---

Oh yeah, about that: It's not mentioned on the website, but the email for attendees of last year's edition contained the following bit:

> While we personally developed many great friendships through having both tracks, we need to acknowledge that we can't provide the same value for our Android Attendees anymore as our friends at droidcon and other conferences. This is why with a heavy heart we decided to drop the Android track this year and focus on what we know best: iOS Development!

---

Oh I didn't know that. Perfect then, thank you @xavierLowmiller!